### PR TITLE
dell-dock: set minimum version of EC

### PR DESCRIPTION
**Pull request by CragW**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by CragW** [comment]
New WD dock is announced, the minimum requirement is therefore added to avoid accidental downgrade to an unsupported version given the WD docks are sharing the same device id.

File: plugins/dell-dock/fu-dell-dock-i2c-ec.c [hunk 1]
```diff
@@ -826,7 +826,6 @@ fu_dell_dock_ec_write_fw(FuDevice *device,
 
 	/* meet the minimum EC version */
 	if ((flags & FWUPD_INSTALL_FLAG_FORCE) == 0 &&
-	    (self->data->module_type != MODULE_TYPE_130_USB4) &&
```
Comment (hughsie) [inline plugins/dell-dock/fu-dell-dock-i2c-ec.c:829]: This is intentional, right?

**Review by hughsie — APPROVED** [review]

**Comment by hughsie** [comment]
> the WD docks are sharing the same device id

:face_with_spiral_eyes:

File: plugins/dell-dock/fu-dell-dock-i2c-ec.c (hunk 1 above)
Comment (CragW) [reply plugins/dell-dock/fu-dell-dock-i2c-ec.c:829]: Yes, it was a quirk for the new dock during development phase.

**Comment by superm1** [comment]
> > the WD docks are sharing the same device id
> 
> :face_with_spiral_eyes: 

They're hardware upgradable

**Comment by superm1** [comment]
> > > the WD docks are sharing the same device id
> > 
> > :face_with_spiral_eyes: 
> 
> They're hardware upgradable 

Sorry didn't finish my thought. So you want to ensure that someone doesn't take an older release in case they switch modules.

**Pull request merged**